tests/__init__.py

```diff
@@ -15,6 +15,9 @@ def setUpClass(cls):
         with open(os.path.join(cls.TEST_DIR, 'payload.txt'), 'rb') as t_file:
             cls.test_data = t_file.read()
 
+        with open(os.path.join(cls.TEST_DIR, 'payloadHTTP.txt'), 'rb') as thttp_file:
```

Comment:
Okay this new file does not seem to be any different than the old one `diff tests/fixtures/payload.txt tests/fixtures/payloadHTTP.txt` is not showing any differences for me.

Reply:
Hmm.... probably a thing that I did not consider in git when adding to the file: https://help.github.com/en/articles/dealing-with-line-endings - so not sure what had happened. Maybe adding a .gitattributes file to the repo? 
Anyway, will be separating this into new branch as suggested and provide more feedback and findings.